### ISSUE TITLE
Internal Error (no-unused-imports)

### ISSUE THREAD

**Issue by allco** [pinterest/ktlint#1050]
The command:

```
# ktlint -a -F **/*.kt
/Users/alsk/Projects/hb_ep/modules/common/src/debug/java/net/easypark/android/extensions/RxExtensions.kt:1:1: Internal Error (no-unused-imports). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
```

The code:
```
package net.easypark.android.extensions

import io.reactivex.Observable

internal fun log(tag: String, value: Any) {
    println("$tag: $value")
}

fun <T> Observable<T>.dump(tag: String): Observable<T> =
    this
        .doFinally { log(tag, "doFinally") }
        .doOnNext { log(tag, "onNext: ${it.toString()}") }
        .doOnError { er -> log(tag, "onError: $er") }
        .doOnSubscribe { log(tag, "onSubscribed") }
        .doOnDispose { log(tag, "onDisposed") }
        .doOnComplete { log(tag, "onComplete") }
```
* Version of ktlint used: 0.40.0
* Operating System and version: MacOS: 10.15.7

**Comment by paul-dingemans** [comment]
Duplicates #996 and will be resolved in version 0.41

**Comment by romtsn** [comment]
Yep, this was already fixed, you could try the snapshot in the meantime
